src/clients/user-client/src/components/navigation.tsx

```
@@ -0,0 +1,102 @@
+            >
+              {item.icon}
+              <p>{item.label}</p>
+            </Link>
+          ))}
+        </div>
+        <div>
+          <button
+            onClick={() => logOutMutation.mutate()}
+            disabled={logOutMutation.isPending}
+            className="text-foreground hover:text-primary/50 flex flex-col items-center justify-center"
+          >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)